.travis-deploy-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Immediately cancel if there is no deployment key
+if [ -z "${STUDIO_TEST_DEPLOY_KEY}" ]; then
+  echo No deployment key. Canceling deployment.
+  exit 0
+fi
+
+set -eu
+
+# Prepare GitHub SSH key
+if [ "${TRAVIS_PULL_REQUEST:-false}" = "false" ]; then
+  srcbranch="${TRAVIS_BRANCH}"
+else
+  srcbranch="pull-request-${TRAVIS_PULL_REQUEST}"
+fi
+echo "${STUDIO_TEST_DEPLOY_KEY}" | base64 -d > ~/.ssh/id_rsa
+chmod 600 ~/.ssh/id_rsa
+ssh-keyscan github.com >> ~/.ssh/known_hosts
+
+set -x
+
+srcpath="$(pwd)"
+deploydir="build-${TRAVIS_REPO_SLUG}-${TRAVIS_BUILD_NUMBER}-${srcbranch}"
+deploydir="$(echo "${deploydir}" | sed 's/[^a-Z0-9]/-/g')"
+export PUBLIC_URL="/${deploydir}"
+npm ci
+npm run build
+
+# Get target repository
+cd
+rm -rf studio-test || :
+git clone "git@github.com:elan-ev/studio-test.git"
+cd studio-test
+git checkout gh-pages
+
+# Add new content
+mv "${srcpath}/build/" "${deploydir}"
+
+# Build new index
+echo '<html><body><ul>' > index.html
+find . -maxdepth 1 -name 'build*' -type d \
+  | sort \
```

Comment:
Could we maybe sort in descending order? That way new builds are at the top. Otherwise we might always have to scroll in the future.

Also: does this break when the build number overflows the 100s digit? Is there an option to tell `sort` to be smart about numbers? Otherwise, can we prepend 0s to the build number?

Reply:
fixed